lib/modules/dosomething/dosomething_api/includes/Transformer.php

```diff
@@ -458,6 +458,7 @@ protected function transformReportbackItem($data) {
   protected function transformUser($data) {
     return array(
       'id' => $data->id,
+      'drupal_id' => $data->drupal_id,
```

Comment:
Oooh okay, my mistake! I thought we wanted this but totally makes sense! Should I just close out this PR?